src/test/java/nextstep/member/application/dto/GithubResponses.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.member.application.dto;
+
+import java.util.Arrays;
+
+public enum GithubResponses {
+    사용자1("aofijeowifjaoief", "access_token_1", "email1@email.com"),
+    사용자2("fau3nfin93dmn", "access_token_2", "email2@email.com"),
+    사용자3("afnm93fmdodf", "access_token_3", "email3@email.com"),
+    사용자4("fm04fndkaladmd", "access_token_4", "email4@email.com"),
+    ;
```

Comment:
Invalid data 도 더미데이터로 관리해주면 좀 더 가독성이 좋아질 것 같습니다 :)

```suggestion
    사용자1("aofijeowifjaoief", "access_token_1", "email1@email.com"),
    사용자2("fau3nfin93dmn", "access_token_2", "email2@email.com"),
    사용자3("afnm93fmdodf", "access_token_3", "email3@email.com"),
    사용자4("fm04fndkaladmd", "access_token_4", "email4@email.com"),
    ACCESS_TOKEN_없는_사용자5("code5", null, "email5@email.com"),
    PROFILE_없는_사용자6("code5", "access_token_non_profile", null)
    ;

Reply:
반영해 보겠습니다.